podman.spec.rpkg

```diff
@@ -83,7 +83,7 @@ Requires: conmon >= 2:2.0.30-2
 %if 0%{?fedora} <= 35
 Requires: containers-common >= 4:1-39
 %else
-Requires: containers-common >= 4:1-46
+Requires: containers-common-extra
```

Comment:
I'm probably too late, someone is probably merging this already, but I just don't understand how you can have an unversioned `Requires`. At some point, `containers-common-extra` is going to get an incompatible change. I don't know what that looks like, but I just have a bad feeling about requiring something without a version.

Reply:
enforced min version on the current EVR of containers-common in the copr.